micromamba/tests/test_activation.py

```diff
@@ -646,75 +576,50 @@ def test_activation(
             ] + evars
             stdout, stderr = call(s)
             res = TestActivation.to_dict(stdout)
-
-            assert find_path_in_str(str(rp / "condabin"), res["PATH"])
-            assert not find_path_in_str(str(rp / "bin"), res["PATH"])
-            assert find_path_in_str(str(rp / "envs" / "xyz"), res["PATH"])
-            assert not find_path_in_str(str(rp / "envs" / "abc"), res["PATH"])
-
-            # long paths
-            s = [
-                f"micromamba create -p {TestActivation.long_prefix} xtensor six -v -y -c conda-forge"
-            ]
-            call(s)
-
-            assert os.path.exists(TestActivation.long_prefix)
-
-            s = [
-                "micromamba activate",
-                f"micromamba activate -p {TestActivation.long_prefix}",
-            ] + evars
-            # currently skipping longprefix _activation_ check with cmd.exe as it fails
-            # on CI for mysterious reasons
-            if not (plat == "win" and interpreter == "cmd.exe"):
-                stdout, stderr = call(s)
-                res = TestActivation.to_dict(stdout)
-
-                assert find_path_in_str(str(rp / "condabin"), res["PATH"])
-                assert not find_path_in_str(str(rp / "bin"), res["PATH"])
-                assert find_path_in_str(TestActivation.long_prefix, res["PATH"])
+            assert find_path_in_str(tmp_root_prefix / "condabin", res["PATH"])
+            assert not find_path_in_str(tmp_root_prefix / "bin", res["PATH"])
+            assert find_path_in_str(tmp_root_prefix / "envs" / "xyz", res["PATH"])
+            assert not find_path_in_str(tmp_root_prefix / "envs" / "abc", res["PATH"])
 
             s = [
                 "micromamba activate",
                 "micromamba activate abc",
-                "micromamba activate xyz --stack",
+                "micromamba activate --stack xyz",
```

Comment:
are we not testing `--stack xyz` vs `xyz --stack` anymore?

Reply:
We were not testing both. We were testing one, now it;'s the other.
Did not need to be changed though...

Reply:
Now we're testing both